docs/src/content/docs/components/card-grids.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: Card Grids
+description: Learn how to wrap multiple cards in a grid.
```

Comment:
Since other descriptions have "in Starlight" I would add that to all descriptions to keep it consistent.
```suggestion
description: Learn how to wrap multiple cards in a grid in Starlight.
```